client/src/app/components/Profile/Profile.tsx

```diff
@@ -1,17 +1,72 @@
 import { useAuth0 } from "@auth0/auth0-react";
+import Loader from "../Loader/Loader";
 
 const Profile = () => {
   const { user, isAuthenticated, isLoading } = useAuth0();
+
   if (isLoading) {
-    return <div>Loading ...</div>;
+    return <Loader />;
   }
 
   return (
     isAuthenticated && (
```

Comment:
no need to check is Authenticated. it will not inside this if its in private route